src/site/content/en/blog/off-main-thread/index.md

```diff
@@ -53,6 +53,14 @@ watch my CDS 2019 talk below.
 
 {% YouTube '7Rrv9qFMWNM' %}
 
+## Why web workers?
+
+JavaScript is, by default, a single-threaded language that runs [tasks](/optimize-long-tasks/#what-is-a-task) on [the main thread](/optimize-long-tasks/#what-is-the-main-thread). However, web workers provide a sort of escape hatch from the main thread by allowing developers to spin up separate threads to handle work off of the main thread. While the scope of web workers is limited and doesn't offer direct access to the DOM, they can be hugely beneficial if there is considerable work that needs to be done that would otherwise overwhelm the main thread.
+
+Where [Core Web Vitals](/vitals/) are concerned, running work off the main thread can be beneficial. In particular, offloading work from the main thread to web workers can reduce contention for the main thread, which can improve important responsiveness metrics such as [Interaction to Next Paint (INP)](/inp/) and [First Input Delay (FID)](/fid/). When the main thread has less tasks to process, it can respond more quickly to user interactions, and process work it _must_ do more quickly.
```

Comment:
```suggestion
Where [Core Web Vitals](/vitals/) are concerned, running work off the main thread can be beneficial. In particular, offloading work from the main thread to web workers can reduce contention for the main thread, which can improve important responsiveness metrics such as [Interaction to Next Paint (INP)](/inp/) and [First Input Delay (FID)](/fid/). When the main thread has less work to process, it can respond more quickly to user interactions.
```